pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -49,6 +56,22 @@ def upsample() -> nn.Upsample:
 
 
 class HourGlassBlock(SequentialWithOutChannels):
+    r"""HourGlassBlock from :cite:`ULVL2016`.
+
+    This block comprises a
```

Comment:
I'm not happy with the text, but I can't pinpoint what it is. What do you think of "This block embeds an ``intermediate`` module between a downsample and upsample operation"?